Tarea6

luyves opened the pull request:
@artcidp , @LeoGalPras  y mía. Faltan los tests.

Commit:
Primer commit de la tarea6

Commit:
Quedó ya la multiplicación

Commit:
Método de potencia

Commit:
Algunos otros métodos que faltaban

Commit:
Algunos cambios de formato y comentarios.

Commit:
Exponencial

Commit:
1/f agregado

Commit:
log incompleto

Commit:
corrijo log

Commit:
más correcciones

LeoGalPras committed:
excluyo la definición de ^,incluyo dos pruebas y anoto otros problemas a resolver

Commit:
FUNCIONA

LeoGalPras committed:
Merge branch 'tarea6' of https://github.com/luyves/2016-2_TSFisicaComputacional into tarea6

Conflicts:
	tareas/yvestruz/Tarea6.ipynb

Commit:
Algunas pruebas y extensión del logaritmo y exp.

Definí la exponencial extendida exp(Taylor,n) y el logaritmo extendido
log(Taylor,n) que hacen la expansión de Taylor del argumento a grado n.

Commit:
Expliqué mejor

LeoGalPras committed:
Merge branch 'tarea6' of https://github.com/luyves/2016-2_TSFisicaComputacional into tarea6

Conflicts:
	tareas/yvestruz/Tarea6.ipynb

LeoGalPras committed:
agrego ^ gral.

LeoGalPras committed:
agrego seno y coseno, aún no funcionan

Commit:
Añado módulo Taylor.jl

LeoGalPras committed:
descripción de la rel. de recurrencia sen y cos

LeoGalPras committed:
Merge branch 'tarea6' of https://github.com/luyves/2016-2_TSFisicaComputacional into tarea6

artcidp committed:
Añadir la función de evaluación de polinomios de Taylor

luyves committed:
Merge pull request #7 from artcidp/tarea6

Añadir la función de evaluación de polinomios de Taylor

LeoGalPras committed:
correcciones a sen y cos ya casi

LeoGalPras committed:
Merge branch 'tarea6' of https://github.com/luyves/2016-2_TSFisicaComputacional into tarea6

Commit:
Cambios a sin y cos

Commit:
Cambios al módulo Taylor.jl para incluir sin y cos

Commit:
Quité el archivo que añadió Arturo por accidente

luyves commented:
En el último commit me refiero a que quité el archivo que Arturo accidentalmente añadió (a la carpeta que no era), no que lo borré yo sin querer. Sólo aclaro.

UriAceves commented:
Puntos menos por ser ambiguo.

``` julia
a = Taylor([0,0,1])
b = Taylor([1,1,1,1])
b/a
```

me da

``` julia
Taylor{Float64}([Inf,Inf,NaN,NaN,NaN,NaN])
```

Es el único problema que he encontrado. Cuando tengan sus test comenten y vuelvo a revisar.

Commit:
Corrijo

luyves commented:
El método tiene problemas al tratar de hacer la serie de Taylor de algo que diverge en x = 0.

Ya corregí el código y, si tomamos
`a  = Taylor([1,0,1,0,0])
`
`b = Taylor([1,1,1,1])`
`b/a`

nos da 

`Taylor([1,1,0,0,0,0,1,1])`
que es consistente con lo que [nos dice Wolfram](http://bit.ly/20FlvnX).
Tomé `a` con más términos 0 de orden superior pues el método lo que hace es truncar la aproximación al rango de a.

Aún faltan los tests, cuando los acabemos comento de nuevo.

Commit:
Actualizo el módulo con la corrección de la división

UriAceves commented:
Sólo un chequeo rápido, ¿qué les da si hacen esto?

``` julia
a = Taylor([0,0,1,0,0])
b = Taylor([0,0,0,0,1])
b/a
```

Commit:
Corrijo división versión 9.121.463

luyves commented:
Tienes razón, no funcionaba. No había vuelta de hoja así que borré la definición que teníamos de la división 1/g e implementé la correcta para f/g (re-utilicé nuestro código así que no fue gran cosa).

Debería de funcionar bien ahora.

artcidp committed:
Test y Módulo actualizado para la tarea 6

Añadí la función de evaluación de polinomios de taylor al módulo,
modifiqué la función == para que pueda comparar polinomios de distinto
grado como Taylor([1,2,0])==Taylor([1,2]). Además, añadí una condición en
el ciclo while de la definición de logaritmo para que ahora pueda evaluar
polinomios de taylor de puros ceros (debe dar -Inf)logaritmo para que
ahora pueda evaluar polinomios de taylor de puros ceros (debe dar -Inf).

artcidp committed:
Corrección de Conflictos

artcidp committed:
Borrar Archivos modificados

artcidp committed:
Merge branch 'tarea6' of https://github.com/luyves/2016-2_TSFisicaComputacional into tarea6

artcidp committed:
Añadir nuevamente los archivos

luyves committed:
Merge pull request #8 from artcidp/tarea6

Tests y Módulo actualizado para la tarea 6

luyves commented:
Ya están los tests.

UriAceves commented:
Bien, buena tarea.

Hago el merge.
